Arrowgene.Ddon.Shared/Entity/PacketStructure/C2LPingReq.cs

```diff
@@ -0,0 +1,24 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class C2LPingReq : IPacketStructure
```

Comment:
Probably good to document either here or in the handler that this is only ever sent once upon successfully logging in / selecting a character?